bin/configs/c-bearer-auth.yaml

```diff
@@ -0,0 +1,4 @@
+generatorName: c
+outputDir: samples/client/others/c/bearerAuth
```

Comment:
can you please add the new folder to the CI workflow so that it will be tested moving forward?

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-c-libcurl-client.yaml

Reply:
Ok, I just pushed that.